src/dotnet-counters-ui/dotnet-counters-ui/ViewModels/CounterGraphViewModel.cs

```diff
@@ -24,9 +24,17 @@ public class CounterGraphViewModel : ReactiveObject, IDisposable
     private readonly ObservableCollection<AddedCounterViewModel> _counters = new();
 
     public PlotModel Model { get; }
-    
+
+    public string Name
+    {
+        get => _name;
+        set => this.RaiseAndSetIfChanged(ref _name, value);
+    }
```

Comment:
I do not see the usage of this property anywhere, nor does VS. Am I missing something?

Reply:
Here https://github.com/kkokosa/dotnet-counters-ui/pull/38/files#diff-4859752bac924c6e5db65be95a95e6f8459a475dcfc3049fc2f39c0f1f0226d1R33

Reply:
Nvm, I found the usage. Some weird VS glitch and me not paying attention.